utils/results/output/securityJobSummary.go

```diff
@@ -46,8 +66,28 @@ func SecurityCommandsJobSummary() (js *commandsummary.CommandSummary, err error)
 	return commandsummary.New(&SecurityCommandsSummary{}, "security")
 }
 
+func CreateCommandSummaryResult(section SecuritySummarySection, cmdResults *results.SecurityCommandResults) (ScanCommandSummaryResult, error) {
+	convertor := conversion.NewCommandResultsConvertor(conversion.ResultConvertParams{})
```

Comment:
```suggestion
	convertor := &conversion.CommandResultsConvertor{}
```
This is redundant, the convertor just takes the params from a ResultConvertParams struct and puts them in a new *CommandResultsConvertor. if you are passing an empty struct you can just create an empty pointer to this struct